pilot/const/function_calls.py

```diff
@@ -664,7 +593,7 @@ def command_definition(description_command='A single command that needs to be ex
                 },
                 'content': {
                     'type': 'string',
-                    'description': 'Full content of the documentation file that needs to be saved on the disk. **IMPORTANT**When you want to add a comment that tells the user to add the previous implementation at that place, make sure that the comment starts with `[OLD CODE]` and add a description of what old code should be inserted here. For example, `[OLD CODE] Login route`.',
+                    'description': 'Full content of the documentation file that needs to be saved on the disk.',
```

Comment:
not sure if removing OLD_CODE requires change in how documentation is created.

Reply:
I tweaked it so it works with the new flow (but there's no need for postprocessing because README is small enough not to trigger OLD CODE).